src/hopen/core.cljc

```diff
@@ -19,25 +19,70 @@
             :else element))]
    (f-eval element)))
 
+;; Top level evaluation functions
+;; The given element can be a "block" to be executed or an simple expression to be evaluated
```

Comment:
You are right that some documentation was needed to describe each function, you did well adding them.

However, I suggest to move them from comments to defn's documentation string. That way, the Clojure's tooling can pick it up.

Example:

```clojure
(defn- my-fn
  "That private function is doing something documented."
  [some-data]
  (... something))
```

Reply:
Updated in 687ec3e